Userland/Applications/CharacterMap/CharacterSearchWidget.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Sam Atkins <atkinssj@serenityos.org>
+ * Copyright (c) 2022-2023, Sam Atkins <atkinssj@serenityos.org>
  * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
Not super important, but I only use that serenityos.org email for copyrights. I use trflynn89@pm.me for git things :)